src/desktop/apps/article/templates/meta.jade

```diff
@@ -14,7 +15,7 @@
 title= title + titleExtension
 meta( name='description', content= search_description )
 link( rel='canonical' href=url )
-if article.featured && article.published
+if article.featured && article.published  && isAMPEligible
```

Comment:
Would it make sense for this to be `article.hasAMP()`?

Reply:
Agreed, we should use the model's `hasAMP` here.